src/components/views/newHunt/NewHunt.js

```diff
@@ -5,20 +5,23 @@ import PropTypes from 'prop-types';
 
 import './newHunt.css';
 
+import { connect } from 'react-redux';
+import { bindActionCreators } from 'redux';
+import { goSubmitHunt } from '../../router/routes';
+import { setActivePage, setManuelSubmit } from '../../../actions';
+
 class NewHunt extends Component {
-  constructor(props) {
-    super(props);
-    this.state = {
-      hideManualSubmit: true,
-    };
+  componentWillMount() {
+    this.props.setActivePage(goSubmitHunt);
   }
 
-  togleHideManualSubmit = () => {
-    return this.setState({ hideManualSubmit: !this.state.hideManualSubmit });
+  toggleHideManualSubmit = () => {
+    const manual = this.props.hideManualSubmit == true ? false : true;
```

Comment:
This could be simplified with `const manual = !this.props.hideManualSubmit`

Reply:
I don't know why but this was an issue where it did not work with the shortened version. Did it work on your version?

Reply:
Yes it does... mmmh :shipit:

Reply:
Anyway, no problem :) It's fixed in my development brach 🚀